frontend/apps/server/src/utils/object.ts

```diff
@@ -0,0 +1,23 @@
+export const manipulateValueOfObject = <T>(
+  targetKey: string,
+  targetObject: T,
+  manipulateFunc: (value: string) => string,
+): T => {
+  // 배열이면 각 요소에 대해 재귀적으로 처리
+  if (Array.isArray(targetObject)) {
+    return targetObject.map(item => manipulateValueOfObject(targetKey, item, manipulateFunc)) as T;
+  }
+  if (!targetObject || typeof targetObject !== 'object') {
+    return targetObject;
+  }
+  const newData: Record<string, unknown> = {};  for (const key in targetObject) {
+    // 만약 target key이면 해당 값에 대해 변환 함수 적용
+    if (key === targetKey) {
+      newData[key] = manipulateFunc(String(targetObject[key]));
+    } else {
+      // 그 외의 값은 재귀적으로 처리
+      newData[key] = manipulateValueOfObject(targetKey, targetObject[key], manipulateFunc);
+    }
+  }
+  return newData as T;
```

Comment:
p3;
`reduce` 고차함수 활용이 더 깔끔해 보입니다!

```suggestion
  return Object.entries(targetObject).reduce((newData, [key, value]) => {
    if (key === targetKey) newData[key] = manipulateFunc(String(value));
    else newData[key] = manipulateValueOfObject(targetKey, value, manipulateFunc);
    return newData;
  }, {} as Record<string, unknown>) as T;
  ```